src/offers/components/OfferForm.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <KFormContainer>
+    <KSpinner v-if="fetching" />
+    <div
+      v-else
+      class="edit-box"
+      :class="{ changed: hasChanged }"
+    >
+      <form @submit.prevent="maybeSave">
+        <QInput
+          v-model="edit.name"
+          :label="$t('OFFER.NAME')"
+          :hint="$t('OFFER.NAME_HELPER')"
+          v-bind="nameError"
+          :autofocus="!$q.platform.has.touch"
+          autocomplete="off"
+          @blur="$v.edit.name.$touch"
+        >
+          <template #before>
+            <QIcon name="fas fa-fw fa-star" />
+          </template>
+        </QInput>
+
+        <MarkdownInput
+          v-model="edit.description"
+          icon="fas fa-fw fa-address-card"
+          :label="$t('OFFER.DESCRIPTION')"
+          v-bind="descriptionError"
+          @keyup.ctrl.enter="maybeSave"
+        />
+
+        <QField
+          v-model="edit.images"
+          v-bind="imagesError"
+        >
+          <template #before>
+            <QIcon name="fas fa-fw fa-star" />
+          </template>
+          <template #control>
+            <MultiCroppa v-model="edit.images" />
```

Comment:
Comment what multicroppa expects to be in v-model, and what it will change/add/etc.